.gitignore

```diff
@@ -1,6 +1,6 @@
 .DS_Store
 .project
 .vagrant/
-config/
-src/
-package-lock.json
```

Comment:
`package-lock.json` is an artifact of VersionBot which will commit it into the repository, so this ignore rule is actually required.

Reply:
I didn't realise, so I will add it back in. Good catch @dfunckt

Reply:
I've added it back, but included the root-directory specifier 👍